docs/tasks/configure-pod-container/configmap.md

```diff
@@ -148,6 +148,65 @@ game.properties:        158 bytes
 ui.properties:          83 bytes
 ```
 
+You can pass in the `--from-env-file` to create a ConfigMap from an env-file, for example:
+
+```shell
+kubectl create configmap game-config-env-file \
+        --from-env-file=docs/tasks/configure-pod-container/game-env-file.properties
+```
+
+would produce the following ConfigMap:
+
+```shell
+kubectl describe configmap game-config-env-file
+Name:           game-config-env-file
+Namespace:      default
+Labels:         <none>
+Annotations:    <none>
+
+Data
+====
+allowed:
+----
+true
+enemies:
+----
+aliens
+lives:
+----
+3
+```
+
+Note that when passing `--from-env-file` multiple times to create a ConfigMap from multiple data sources, only the last env-file is used:
+
+```shell
+kubectl create configmap config-multi-env-files \
+        --from-env-file=docs/tasks/configure-pod-container/game-env-file.properties \
+        --from-env-file=docs/tasks/configure-pod-container/ui-env-file.properties
+```
+
+would produce the following ConfigMap:
+
+```
+kubectl describe configmap config-multi-env-files
```

Comment:
can we consider showing the output from `kubectl get configmap config-multi-env-files -o yaml` instead. it would be a lot intuitive.